usr/lib/common/mech_aes.c

```diff
@@ -488,6 +488,107 @@ CK_RV aes_ctr_decrypt(STDLL_TokData_t *tokdata,
     return rc;
 }
 
+CK_RV aes_xts_encrypt(STDLL_TokData_t *tokdata,
+                      SESSION *sess,
+                      CK_BBOOL length_only,
+                      ENCR_DECR_CONTEXT *ctx,
+                      CK_BYTE *in_data,
+                      CK_ULONG in_data_len,
+                      CK_BYTE *out_data, CK_ULONG *out_data_len)
+{
+	OBJECT *key = NULL;
```

Comment:
please do NOT use tabs for indention. Use 4 blanks instead of 1 tab. (throughout the whole function)